docs/community/tutorials-contest/2021/NFT-Marketplace-on-Avalanche/README.md

```diff
@@ -80,7 +80,7 @@ We will be using [React JS](https://reactjs.org) to build the frontend of our NF
 - [Metamask](https://metamask.io) extension must be installed on your browser.
 - [create-react-app](https://www.npmjs.com/package/create-react-app) must be installed.
 
-# Getting Started
+## Getting Started
```

Comment:
This is not the right transformation to perform. We should then make all children #+1.